Sources/ProjectDescription/Config.swift

```diff
@@ -6,6 +6,16 @@ public typealias TuistConfig = Config
 public struct Config: Codable, Equatable {
     /// Contains options related to the project generation.
     public enum GenerationOptions: Codable, Equatable {
+        /// Contains options for code coverage
+        public enum CodeCoverageMode: Codable, Equatable {
+            /// Gather code coverage for all targets
+            case all
+            /// Gather code coverage only for targets, that have code coverage enabled at least in one scheme
+            case relevant
+            /// Gather code coverage only for given targets
+            case targets([TargetReference])
```

Comment:
@fortmarek / @kwridan do you foresee any issues having references from the `Config.swift` to the graph? I realized it's the first time we do something like this but I can't think of any issue.

Reply:
@olejnjak I'd extend the linting phase of the generation to ensure the targets referenced from here are present in the project.